### ISSUE TITLE
Support EVM tracing

### ISSUE THREAD

**Issue by MOZGIII** [humanode-network/humanode#797]
This is not yet implemented in the frontier, but Moonbeam has implemented this in their RPC: https://github.com/moonbeam-foundation/moonbeam/tree/d47da4eee70f60bd835c39f9e6098e1e4e0c9347/client/rpc/debug

**Comment by dmitrylavrenov** [comment]
Already implemented https://github.com/polkadot-evm/frontier/pull/1237.

We need our forks to be updates to `polkadot-v1.1.0`.

**Comment by MOZGIII** [comment]
I don't think that one adds tracing actually

**Comment by MOZGIII** [comment]
Yep, we'd actually have to port the implementation from moonbeam or somewhere else, also tracing support comes with a unique challenge of working with multiple runtime versions - one for usual work and one for tracing (compile with certain compile-time features enables that are required for tracing but slow down the execution too much to be used for the usual chain operation)
